src/zenml/cli/stack.py

```diff
@@ -564,33 +564,45 @@ def describe_stack(stack_name: Optional[str]) -> None:
 
 @stack.command("delete")
 @click.argument("stack_name", type=str)
-def delete_stack(stack_name: str) -> None:
+@click.option("--yes", "-y", is_flag=True, required=False)
+def delete_stack(stack_name: str, yes: bool = False) -> None:
     """Delete a stack."""
     cli_utils.print_active_profile()
+    if yes:
```

Comment:
```
confirmation = yes or cli_utils.confirmation(
            f"This will delete stack '{stack_name}' from your repository. \n"
            "Are you sure you want to proceed?"
        )
```